src/herder/HerderSCPDriver.cpp

```diff
@@ -711,22 +723,82 @@ HerderSCPDriver::combineCandidates(uint64_t slotIndex,
                 {
                     highestTxSet = cTxSet;
                     highest = sv.txSetHash;
+                    if (candidateCloseTimes.size() == 0)
```

Comment:
this cannot happen, a few lines up you're inserting into the vector

Reply:
Alright, another `logic_error` bites the dust.  :)